### PR TITLE
K8s support

### LINKED ISSUE #281: Create the required manifests to deploy the demo to a k8s cluster
In a k8s folder we want to create the required manifests and docs for deploying the existing solution to a k8s cluster.
We want to use the same backend/frontend Dockerfile and push the required containers to a public accessible registry.

### PR DESCRIPTION
**Pull request by andreacfm**
close #281

### CODE REVIEW

File: Backend.dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ COPY pkg-svcs/backend/ /app/backend/
 
 FROM source as builder
 WORKDIR /app/backend
-RUN mv .env.registry .env
```
Comment (andreacfm) [inline Backend.dockerfile:21]: @heidermassu I do not think we need this change. We will override any env var from the k8s service manifest

File: K8s/manifest/collector-works.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+apiVersion: opentelemetry.io/v1alpha1
```
Comment (andreacfm) [inline K8s/manifest/collector-works.yaml:1]: Let's rename it as collector.yml. We can drop the old/* stuff.

File: K8s/manifest/collectorconfig.yaml [hunk 3]
```diff
@@ -0,0 +1,143 @@
+---
```
Comment (andreacfm) [inline K8s/manifest/collectorconfig.yaml:1]: We should not need this anymore. We are deploying the collector with the correct configuration directly in the OpenTelemetryCollector resource

File: K8s/manifest/collectorconfig_OLD [hunk 4]
```diff
@@ -0,0 +1,142 @@
+---
```
Comment (andreacfm) [inline K8s/manifest/collectorconfig_OLD:1]: Let's drop this one.

File: K8s/manifest/jaeger.yaml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      protocol: TCP
+      port: 14269
+    - name: port-9411
+      protocol: TCP
+      port: 9411
+    - name: otlp-grpc
+      port: 4317
+    - name: otlp-http
+      port: 4318
+  type: LoadBalancer
```
Comment (andreacfm) [inline K8s/manifest/jaeger.yaml:82]: We should revert this at some point. Maybe let's comment this out.

Comment (heidermassu) [reply K8s/manifest/jaeger.yaml]: I've pushed a new version of this file where we have 2 services and just port 16686 is as LoadBalancer the other one (14250, 4317, 4318) are as ClusterIP (only internal)

File: Backend.dockerfile (hunk 1 above)
Comment (heidermassu) [reply Backend.dockerfile:21]: Line removed. That worked.

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,26 @@
+IMAGE_NAME := otlp-blueprint
+GITHUB_REPOM := heidermassu/otlp-blueprint
```
Comment (andreacfm) [inline Makefile:2]: Does it work using the NearForm repo registry?

**Pull request merged**